src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
```

Comment:
@valib All the `case "oulf"` should be renamed, is that correct ?
Sorry, i saw that only later.